Development about faq

sgr2848 opened the pull request:
This pull request has migrated pages of Landing, About, FAQ and Teams to nextjs.

KevinTamCS committed:
Create FUNDING.yml

sgr2848 committed:
Merge branch 'development' of https://github.com/hack-brooklyn/hackbrooklyn.org

sgr2848 committed:
Migrated Landing Page to next.js

sgr2848 committed:
Finished about & faq migration to next js

sgr2848 committed:
Migrated teams page

sgr2848 committed:
All the pages migrated to next js

KevinTamCS commented on src/pages/index.js:
```diff
@@ -1,65 +1,21 @@
+    return <HomePage />
+    // return (ReactDOM.render(<HomePage />, root_container));
 }
+// const root = get_root();
```
I don't think setting the root via `useEffect()` is a good idea. If the page gets server-side rendered, `useEffect()` does not get triggered and the default value of `null` will be used instead.

KevinTamCS reviewed (changes requested):
Most of the Next.js migration looks good! Just a few things that caught my eye:
- For SCSS module class names, the recommended convention that we're following is to use `camelCase` instead of `snake_case`.
- The Apply button should probably not be using the Bootstrap success button style.
- Next.js actually has a separate root ID called `__next`, we should remove references to the `root` ID. Setting the document root via `useEffect()` may also cause issues with SSR.

Other than that, the landing page looks great! Great work on it!

sgr2848 committed:
Changed the button from Bootstrap to pre-existing style

sgr2848 committed:
Changed scss class name to conventional camelCase naming convention

sgr2848 committed:
Changed the root id from 'root' to __next and removed the use of useEffect in index page

KevinTamCS commented on src/pages/index.js:
```diff
@@ -7,15 +7,15 @@ import HomePage from './HomePage';
+        console.log("Log 1")
+        return (ReactDOM.render(<HomePage />, root_document));
+    };
+
```
I think we could just remove the entire root setting since Next.js handles the rendering for us.

The final thing should look something like this:

```jsx
const IndexPage = () => {
    return <HomePage />
}
```

KevinTamCS reviewed (changes requested):
Much nicer! Just a few more tweaks to match the original site and we can merge.

Refer to [https://hackbrooklyn.org](https://hackbrooklyn.org) to see what the final site should look like.

- Refer to the comment below about the `root` node setting.
- Landing Section:
  - Change the Apply Now `<button>` to an `<a>` since it's a link
  - Fix button outline to match the original page (if changing to an `<a>` didn't fix it)
  - How it looks now:
  ![image](https://user-images.githubusercontent.com/28375751/110515402-5e658400-80d6-11eb-988b-308587a31b05.png)
  - How it should look:
  ![image](https://user-images.githubusercontent.com/28375751/110515428-658c9200-80d6-11eb-8085-bd25e6bd08a5.png)
- FAQ Section:
  - Title should be centered
  - Line spacing should match the original page
  - How it looks now:
	![image](https://user-images.githubusercontent.com/28375751/110514346-33c6fb80-80d5-11eb-959a-fedcdd76eec0.png)
  - How it should look:
	![image](https://user-images.githubusercontent.com/28375751/110514437-48a38f00-80d5-11eb-9b74-23fa59fbb113.png)
- Organizers Section:
  - Font family for the headers should be `"Nunito", sans-serif`
  - Add more padding to the sides of the organizers and contributors
  - Add some margin/padding between the organizer and contributor photos and their names
  - How it looks now:
	![image](https://user-images.githubusercontent.com/28375751/110514757-a041fa80-80d5-11eb-9d42-88552332b7e0.png)
  - How it should look:
  ![image](https://user-images.githubusercontent.com/28375751/110514709-93bda200-80d5-11eb-9139-36d3123686bf.png)
- Sponsors Section:
  - Add line spacing between the message
  - Change the Contact Us `<button>` to an `<a>` since `mailto:` is a link
  - Fix button outline to match the original page (if changing to an `<a>` didn't fix it)
  - How it looks now:
  ![image](https://user-images.githubusercontent.com/28375751/110515620-9240a980-80d6-11eb-82d4-18c1018eab37.png)
  - How it should look:
  ![image](https://user-images.githubusercontent.com/28375751/110515042-f020c180-80d5-11eb-8a1d-c432bc76a272.png)

Commit:
Completed requested changes

 Fixed both buttons, removing their outlines (left as buttons as changing to an a tag did not auto work). Centered faq title, and gave appropiate line spacing for the faqanswer. Changed the font-family for the organizers section and adding the requested padding/margin. Added line spacing issue in the sponsor section and also fixed the button as the landing section's button.

KevinTamCS committed:
Fix organizer year heading font

KevinTamCS committed:
Fixed button hovers

KevinTamCS commented:
Made some slight changes but everything else looks good for now!

KevinTamCS reviewed (approved):
Made some slight changes but everything else looks good for now! Great job @sgr2848 and @GezimHakramaj!
